Formula/perl.rb

```diff
@@ -30,6 +31,7 @@ def install
       -Duseshrplib
       -Duselargefiles
       -Dusethreads
+      -Dsed=/usr/bin/sed
```

Comment:
Maybe we need to add `uses_from_macos "sed"` then?

Reply:
We do not have a `sed` formula.

This line could be excluded on Linux since we only have a superenv shim for sed on macOS.

Reply:
Right, I've confused it with `gnu-sed`. Sorry for noise

Reply:
I guess the Linux folk assume system sed exists and is in a consistent location. I'm not entirely sure, but I don't see much `gnu-sed` usage in Linuxbrew/core.